src/main/java/edu/kis/powp/jobs2d/command/RotateCommand.java

```diff
@@ -0,0 +1,60 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class RotateCommand implements CommandVisitorInterface {
```

Comment:
The name of the class is confusing. It is not a command but a utility to perform rotation. Similarly other visitor names are to be fixed

Reply:
Fixed